components/MapCard.vue

```diff
@@ -174,6 +174,7 @@ export default {
 <style scoped lang="scss">
 .v-slider__tick-label {
   transform: rorate(90deg);
+  font-size: 11px;
```

Comment:
こんな感じだとよりそれっぽくなりそうです！ 👍 

```suggestion
    @include font-size(11);

    transform: translateX(-50%) translateY(30%) rotate(320deg);
    color: $gray-3;
```

Reply:
`transform` が効かなくてこれだとダメですね...すみません。
僕もちょっと調査してたんですが、fontsizeだけだと320pxの端末で重なっちゃうぽいので、斜めにしたほうがよいかもです！w

参考まで！
https://github.com/kawokas/covid19/blob/4440f48e49ddfbdf759df49df96fd2678b98cd4a/components/MapCard.vue#L188